css-box-model/styles.css

```diff
@@ -0,0 +1,7 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin-left: 12px;
+  margin-right: 12px;
+  padding: 6px 16px;
+}
```

Comment:
Please set the radius of the button's border:

---

![image](https://user-images.githubusercontent.com/16143771/176504986-d39cf14d-9ce7-427e-b2bf-d32af83e279a.png)

Reply:
Added the border radius!